### ISSUE TITLE
Automatically generate CLI shortcuts

### ISSUE THREAD

**Issue by nmattia** [nmattia/niv#71]
The [ongoing niv refactor](https://github.com/nmattia/niv/pull/68/files#diff-ed2831e1e73d8e870c55a05363330406R72) allows tracking which keys are accessed statically. That means that [“shortcut attributes”](https://github.com/nmattia/niv/pull/68/files#diff-7102eb1d0519ad358f3a737e8b762c5bR136) can be inferred automatically.

**Comment by andir** [comment]
@nmattia can you rephrase this? What are "shortcut attributes"? The link you provided just points to the refactoring commit.

**Comment by nmattia** [comment]
That means that `niv update --help` could infer from the applicative _which_ keys are read during the update (like `sha256`, `url`, etc) and could automatically provide command line flags for them (`--sha256`, etc).
